packages/browser/src/index.ts

```diff
@@ -0,0 +1,70 @@
+import Rum from '@hyperdx/otel-web';
+import SessionRecorder from '@hyperdx/otel-web-session-recorder';
+
+const URL_BASE = 'https://otel-in.hyperdx.io';
+
+function hasWindow() {
+  return typeof window !== 'undefined';
+}
+
+class Browser {
+  init({
+    url,
+    apiKey,
+    service,
+    tracePropagationTargets,
+    maskAllText = false,
+    maskAllInputs = true,
+  }: {
+    url?: string;
+    apiKey: string;
+    service: string;
+    tracePropagationTargets?: (string | RegExp)[];
+    maskAllText?: boolean;
+    maskAllInputs?: boolean;
+  }) {
+    if (!hasWindow()) {
+      return;
+    }
+
+    const urlBase = url ?? URL_BASE;
+
+    Rum.init({
+      url: `${urlBase}/v1/traces`,
+      allowInsecureUrl: true,
```

Comment:
wonder if we can disable this ??

Reply:
I guess we can surface it to the user if we really wanted, I'm not 100% sure there is much security merit to having a flag like this tbh though which is why I set it to be true for now.